### PR TITLE
Wilson/modal dynamic campaign text

### PR DESCRIPTION
**Pull request by wilsonj806**
Update GAM modal to render the campaign distributor off of API data rather than hard coded data. It fetches the distributor for the campaign that expires the soonest(didn't use the distributor that was included with the `getSeller()` API utility function since it didn't include distributor URL)

### CODE REVIEW

File: src/components/ModalBuyMealNew/ModalBuyMealPool.tsx [hunk 1]
```diff
@@ -45,8 +48,34 @@ export const Modal = (props: Props) => {
...
+        <a
+          href={campaignDistributor.website_url}
+          target="_blank"
+          rel="noopener noreferrer"
+          className={styles.link}
+        >
+          {' '}
+          {campaignDistributor.name}
+        </a>
+        { ' '}
```
Comment (jacoblb64) [inline src/components/ModalBuyMealNew/ModalBuyMealPool.tsx:76]: nit: extra space here?

File: src/components/ModalBuyMealNew/ModalBuyMealPool.tsx [hunk 2]
```diff
@@ -59,35 +88,7 @@ export const Modal = (props: Props) => {
...
-        <a
-          href="https://www.cpc-nyc.org/"
-          target="_blank"
-          rel="noopener noreferrer"
-          className={styles.link}
-        >
-          {' '}
-          the Chinese-American Planning Council{' '}
-        </a>
+        { Distributor }
```
Comment (jacoblb64) [inline src/components/ModalBuyMealNew/ModalBuyMealPool.tsx:91]: I think if you want, you can change `Distributor` to a lambda that returns that body, and add it as a component here like `<Distributor></Distributor>` -- up to you

**Review by jacoblb64 — APPROVED** [review]
Looks good! Added some non-blocking comments. I don't think we have any plans at the moment to run multiple campaigns for a merchant at any one time, but I think when we do we should update this to show all the distributors for active campaigns instead of just the latest.

**Comment by wilsonj806** [comment]
Gotcha, that shouldn't be too bad. Should probably also check to make sure the campaign is active before listing it's distributor also.

**Pull request merged**